docker/all-in-one/Dockerfile

```diff
@@ -65,7 +65,7 @@ FROM supabase/gotrue:v${gotrue_release} as gotrue
 ####################
 FROM base as kong
 ARG kong_release
-ADD "https://download.konghq.com/gateway-2.x-ubuntu-focal/pool/all/k/kong/kong_${kong_release}_${TARGETARCH}.deb" \
+ADD "https://packages.konghq.com/public/gateway-28/deb/ubuntu/pool/focal/main/k/ko/kong_${kong_release}/kong_${kong_release}_${TARGETARCH}.deb" \
```

Comment:
Maybe adding a comment close to `ARG kong_release=2.8.1` to warn to verify this URL before upgrading above `2.8.x`, as the gateway url changes?

Reply:
Good idea, done. Though hopefully we can remove Kong entirely sometime.